src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -184,6 +189,26 @@ static MvcCoreLoggerExtensions()
                 5,
                 "Could not find an output formatter based on content negotiation. Accepted types were ({AcceptTypes})");
 
+            _inputFormatterSelected = LoggerMessage.Define<IInputFormatter, string>(
+                LogLevel.Debug,
+                1,
+                "Selected input formatter '{InputFormatter}' for content type '{ContentType}'");
```

Comment:
Some log msgs end in periods, some don't. Let's at least make the new one consistent (presumably by adding a period).